pkg/driver/type/kmod.go

```diff
@@ -223,8 +215,14 @@ func (k *kmod) Build(ctx context.Context,
 			printer.Logger.Warn("Could not fill /tmp/falco-dkms-make content.")
 			continue
 		}
-		dkmsCmdArgs := fmt.Sprintf(`dkms install --directive="MAKE='/tmp/falco-dkms-make'" -m %q -v %q -k %q --verbose`,
-			driverName, driverVersion, kr.String())
+		var dkmsCmdArgs string
+		if kernelDir, found := env[KernelDirEnv]; found {
+			dkmsCmdArgs = fmt.Sprintf(`dkms install --kernelsourcedir %q --directive="MAKE='/tmp/falco-dkms-make'" -m %q -v %q -k %q --verbose`,
```

Comment:
If `KERNELDIR` env var was passed, forward it to `dkms install` too!

Reply:
Note: `ebpf` did not need any change since it was already passing all the env to the `make` command.